addons/hide-show-context-menu/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Hide sprites via context menu",
```

Comment:
```suggestion
  "name": "Hide sprites from context menu",
```

Reply:
This sounds to me like regularly, the sprites would appear in a context menu, and this addon hides them from it.

Reply:
```suggestion
  "name": "Hide sprites with context menu",
```